### ISSUE TITLE
I can't run it :<(

### ISSUE THREAD

**Issue by Myoontyee** [casparwylie/Perceptron#4]
Hello! Your work are very amazing!  
But I meet a question like this :

 tk_main = Tk()
NameError: name 'Tk' is not defined

Could you teach me how to deal with it?
THX!

**Comment by Myoontyee** [comment]
we use mttkinter or Tkinter??

**Comment by casparwylie** [comment]
Yep, install the mtTkinter file in this repo https://github.com/RedFantom/mtTkinter, then import that instead of Tkinter.
